### ISSUE TITLE
MongoDB 4 support?

### ISSUE THREAD

**Issue by carstenbaumhoegger** [prisma/Mongo-Connector-Preview#3]
Hey!
Currently, the Mongo connector uses MongoDB `3.6`, although `4.0` came out earlier this year with many useful features.
Is support for the newer version planned?

**Comment by do4gr** [comment]
Yes, that is certainly planned. I'll check whether it just works out of the box next week.

**Comment by do4gr** [comment]
I did a cursory check and it seems to just work. You can just change the Mongo version in the docker-compose.yml from  `image: mongo:3.6` to `image: mongo:4.0`. When you then restart the containers with `docker-compose up -d` it will pull the Mongo 4.0 image and work against that. 

I think the most interesting feature of 4.0 is transactions, but these would need support and code changes from our side to take effect. But you might reap the benefits of some under the hood improvements in Mongo when using 4.0. 

Let me know if you are using/testing 4.0 a little more in depth and run into problems.

**Comment by firrae** [comment]
So far no issues for me @do4gr, though I'm not doing anything too crazy.

**Comment by do4gr** [comment]
Good to know, thanks for reporting back @firrae
